src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineView.xaml.cs

```diff
@@ -282,7 +288,8 @@ private void SymbolTreeItem_SourceUpdated(object sender, DataTransferEventArgs e
                 {
                     var textView = _viewTracker.GetActiveView();
                     textView.TryMoveCaretToAndEnsureVisible(
-                        symbolModel.Data.SelectionRangeSpan.TranslateTo(textView.TextSnapshot, SpanTrackingMode.EdgeInclusive).Start);
+                        symbolModel.Data.SelectionRangeSpan.TranslateTo(textView.TextSnapshot, SpanTrackingMode.EdgeInclusive).Start,
+                        _outliningManagerService);
```

Comment:
how does this work, does this helper just know to expand in that case? 

note: if so, it seems like a super weird helper since we'd always want htis behavior, in which case we should always require the outlining manager.  but this is ok for now.

Reply:
This was exactly my conclusion